RevenueCatUI/Data/Localization.swift

```diff
@@ -31,7 +31,7 @@ enum Localization {
         return self.unitAbbreviationLengthPriorities
             .lazy
             .compactMap { length in options.first { $0.count == length } }
-            .first
+            .first { _ in true } // See https://github.com/apple/swift/issues/55374
```

Comment:
Should we create our own extension `firstElement` or something like that for this?

Reply:
We'd need to duplicate it in both `RevenueCat` and `RevenueCatUI` (since we can't use the `package` access type yet).